### DISCUSSION TITLE
FL Studio 2024

### DISCUSSION THREAD

**Discussion by scarcekoi** [catppuccin/catppuccin#2622] [Port Requests]
### Will you make the port?

- [x] Yes, I'm willing to make the port

### How can the suggested port be themed?

![image](https://github.com/user-attachments/assets/a1c86331-a98d-42a1-9818-155dd9c47e34)
By changing those colours.

### Any additional comments?

It would be similar to [Dark Reader](https://github.com/catppuccin/dark-reader)'s instructions.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by scarcekoi** [comment]
Also, in `C:\Program Files\Image-Line\FL Studio 2024\Artwork\Themes` there are .flstheme files which can also be done. E.g:
light tangerine.flstheme
```flstheme
Hue=0
Saturation=-251
Lightness=157
Contrast=64
Text=-226
Selected=-31679
Highlight=-16746778
Mute=-16749326
Option=-16749326
StepEven=-4276805
StepOdd=-15043097
Lightmode=1
OverrideClips=1
TextColor=-11319997
Meter0=-16747542
Meter1=-13594113
Meter2=-10775297
Meter3=-12745217
Meter4=-7028993
Meter5=-1510657
NoteColor0=-15500033
NoteColor1=-16411650
NoteColor2=-11954691
NoteColor3=-15298817
NoteColor4=-13726721
NoteColor5=-13129217
NoteColor6=-10119685
NoteColor7=-16485121
NoteColor8=-11887617
NoteColor9=-9520641
NoteColor10=-9001481
NoteColor11=-16750616
NoteColor12=-8735233
NoteColor13=-7877377
NoteColor14=-6896643
NoteColor15=-16752941
PRGridback=5129268
PRGridCustom=0
PRGridContrast=100
PLGridback=-1513240
PLGridCustom=1
PLGridContrast=200
EEGridback=5129268
EEGridCustom=0
EEGridContrast=100
BackMode=2
BackPicFilename=FireBG.png
BackHTMLFileName=Default.txt
BackColor=-16777216
```

**Comment by yhqe** [comment]
bumping this, would be a really cool add to my setup

**Comment by eszn** [comment]
bump
